test/functionalTest/cases/0647_crash_with_compounds/POST_UpdateContextRequest.test

```diff
@@ -0,0 +1,148 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This (an a couple more of .test files in this PR) are not actually new files but old ones adapted. However, it seem that there has been so many changes and github doesn't detect them as "moved" but "delete file" + "new file".

NTC (just a remark).